app/assets/javascripts/components/physical_infrastructures/physical-server-toolbar.js

```diff
@@ -0,0 +1,50 @@
+ManageIQ.angular.app.component('physicalServerToolbar', {
+  bindings: {
+    physicalServerId: '=?',
+  },
+  controllerAs: 'vm',
```

Comment:
Oh, in that case we surely should. As we don't call model instances "model" in Rails code or we don't call bool variables "bool" or "boolean" or in pascal we don't call a string variable a "string" unless we are writing a tutorial on using different types and want to point out the difference ;-)